app/assets/javascripts/services/project_roles_svc.js

```diff
@@ -0,0 +1,27 @@
+samson.service('projectRolesService', function($http, $q) {
+
+  this.loadProjectRoles = function() {
+    return $http.get('/project_roles');
+  };
+
+  this.createProjectRole = function(project_role) {
+    var deferred = $q.defer();
+    $http.post('/projects/' + project_role.project_id + '/project_roles', project_role.buildCreatePayload()).then(
+      function(response) {
+        project_role.id = response.data.id;
+        deferred.resolve();
+      },
+      function() {
+        deferred.reject();
+      }
+    );
+    return deferred.promise;
+  };
+
+  this.updateProjectRole = function(project_role) {
+    return $http.put('/projects/' + project_role.project_id + '/project_roles/' + project_role.id,
+      project_role.buildUpdatePayload()
```

Comment:
Well, regarding this one and the next comment bellow, I think it could go both ways:
- I agree that the service is the one who knows which parameters it needs to send, but...
- The UserProjectRole is the one that actually knows which fields it supports, reason why I was asking it in each scenario :)

But in the end, I guess it makes it more explicit if, in the service, you can see which fields are being sent to the backend, so, I'll move this to the service. ;)